python/lsst/summit/extras/focusAnalysis.py

```diff
@@ -344,7 +345,7 @@ def fitDataAndPlot(self, hideFit=False, hexapodZeroPoint=0):
 
             quadFitPars = np.polyfit(focusPositions, widths, 2)
             if not hideFit:
-                axes[1].plot(fineXs, np.poly1d(quadFitPars)(fineXs), c=self.COLORS[spectrumSlice])
+                axes[1].plot(fineXs, Polynomial(quadFitPars)(fineXs), c=self.COLORS[spectrumSlice])
```

Comment:
Polynomial and poly1d have reversed order of coefficients, so I don't think this would work (but I don't know the origin of quadFitPars etc).  See https://numpy.org/doc/stable/reference/routines.polynomials.html#transition-guide